app/modules/sccmstatus/sccm_status_model.php

```diff
@@ -0,0 +1,69 @@
+<?php
+class sccm_status_model extends Model {
+
+        function __construct($serial='')
+        {
+                parent::__construct('id', 'sccm_status'); //primary key, tablename
+                $this->rs['id'] = '';
+                $this->rs['serial_number'] = $serial; $this->rt['serial_number'] = 'VARCHAR(255) UNIQUE';
+                $this->rs['agent_status'] = '';
+                $this->rs['mgmt_point'] = '';
+                $this->rs['enrollment_name'] = '';
+                $this->rs['enrollment_server'] = '';     
+                $this->rs['last_checkin'] = '';
+                $this->rs['cert_exp'] = '';       
+
+                // Schema version, increment when creating a db migration
+                $this->schema_version = 0;
+                
```

Comment:
You need to add indexes for columns you are going to need to search or sort on. Which usually is every column apart from id and serial_number.